_sass/salween-theme.scss

```diff
@@ -169,7 +169,7 @@ h1.navbar-header {
     padding: 50px 0 50px;
     color: $base-color-white;
     text-align: center;
-    background: #1988BD url("./assets/Images/BG-3.png") no-repeat left top;
+    background: #1988BD url("../images/BG-3.png") no-repeat left top;
```

Comment:
on the font file use this `src: url('../fonts/OpenSans-LightItalic-webfont.eot');`